autofill/autofill.py

```diff
@@ -28,12 +29,23 @@
     help="Create a PDF export of the card images and do not create a project for MPC.",
     is_flag=True,
 )
-def main(skipsetup: bool, browser: str, exportpdf: bool) -> None:
+@click.option(
+    "--preventsleep",
+    default=False,
+    help="Prevents the system from falling asleep during execution",
+    is_flag=True,
+)
+def main(skipsetup: bool, browser: str, exportpdf: bool, preventsleep: bool) -> None:
     try:
+        if preventsleep:
```

Comment:
it looks like wakepy has a context manager syntax which i think would be preferable (then you can remove the `unset_keepawake` call below):
```suggestion
        with keepawake(preventsleep):
```